mongo-tableau-connector-downloads_template.json

```diff
@@ -0,0 +1,10 @@
+{
+  "manual_link": "https://www.mongodb.com/docs/atlas/data-federation/query/query-with-sql/",
+  "release_notes_link": "https://www.mongodb.com/docs/atlas/data-federation/query/sql/release-notes/",
+  "versions": [
+    {
+      "version": "{RELEASE_VERSION}",
```

Comment:
Wanted to double check that `platform` does not need to be specified here?  As in versions -> platform -> download_link.
Not sure how the download center parses it but the 2 example I see in `mongodb-bi-downloads.json` and the SQL-1358 PR so.

Reply:
There is no platform for a taco file, right?
I shared templates of the json files with the team responsible for the download centre integration and did not hear anything from them. I'll make sure to keep an eye on this.

Reply:
> There is no platform for a taco file, right?  

That's right, it's the same taco file used for Mac/Windows.

> I shared templates of the json files with the team responsible for the download centre integration and did not hear anything from them.

Cool, sounds good then if they've already checked it.